launchers/snesmsu1

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash
 romname="${@%.zip}"
-cd .snesmsu1
+#cd .snesmsu1
```

Comment:
why is this being commented out? Is it an accidental change?

Reply:
No just habit. The command isn't doing anything anyway since it already starts in that directory so I commented it out for cleanliness. I could just delete it also.  The pull is primarily to add exec to the file.